From: =?utf-8?q?Sandro_Kanu=C3=9F?= <hefee@debian.org>
Date: Thu, 25 Jan 2018 10:02:33 -0300
Subject: Make kauth-gen-policy useful for cross compilation

kauth-gen-policy appears to be a tool for turning a text file in
qsettings format (similar to .ini) into an xml file. Both are textual,
so it seems like putting kauth-gen-policy into a Multi-Arch: foreign
package (like kconfig_compiler_kf5 in #887452). Thus we can draw the
same conclusion.

Origin: Debian
Bug-Debian: https://bugs.debian.org/887521
Last-Update: 2018-01-21
---
 CMakeLists.txt |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -48,7 +48,7 @@ ecm_setup_version(PROJECT VARIABLE_PREFIX KAUTH

 find_package(KF6CoreAddons ${KF_DEP_VERSION} REQUIRED)

-set(KAUTH_HELPER_INSTALL_DIR "${KDE_INSTALL_LIBEXECDIR_KF}/kauth")
+set(KAUTH_HELPER_INSTALL_DIR "${KDE_INSTALL_LIBEXECDIR_KF}/kauth" CACHE STRING "Set directory for kauth helper tool")

 if(IS_ABSOLUTE ${KAUTH_HELPER_INSTALL_DIR})
     set(KAUTH_HELPER_INSTALL_ABSOLUTE_DIR ${KAUTH_HELPER_INSTALL_DIR})

